compiler/modules/print.llvm/llvm.bal

```diff
@@ -1031,6 +1022,88 @@ function escapeIdentChar(string:Char ch) returns string {
     }
 }
 
+function gepBody(Value ptr, Value[] indices, "inbounds"? inbounds, "constantExp"? constantExp=()) returns [(string|Unnamed)[],PointerType] {
+    (string|Unnamed)[] words = [];
+    words.push("getelementptr");
+    if inbounds != () {
+        words.push(inbounds);
+    }
+    if constantExp != () {
+        words.push("(");
+    }
+    Type ptrTy = ptr.ty;
+    if ptrTy is PointerType {
+        words.push(typeToString(ptrTy.pointsTo));
+    } else {
+        panic err:illegalArgument("GEP on non-pointer type value"); 
+    }
+    words.push(",", typeToString(ptr.ty), ptr.operand);
+    Type resultType = ptr.ty;
+    int resultAddressSpace = 0;
+    foreach var index in indices {
+        words.push(",");
+        words.push(typeToString(index.ty));
+        words.push(index.operand);
+        if resultType is PointerType {
+            resultAddressSpace = resultType.addressSpace;
+            resultType = resultType.pointsTo;
+        } 
+        else {
+            if resultType is ArrayType {
+                resultType = resultType.elementType;
+            } 
+            else if resultType is StructType {
+                int i;
+                if index.operand is Unnamed {
+                    i = <Unnamed>index.operand;
+                } else {
+                    i = checkpanic int:fromString(<string>index.operand);
+                }
+                if index.ty != "i32" {
+                    panic err:illegalArgument("structures can be index only using i32 constants"); 
+                } 
+                else {
+                    resultType = getTypeAtIndex(resultType, i);
+                }
+            } 
+            else {
+                panic err:illegalArgument(string `type  ${typeToString(resultType)} can't be indexed`);
+            }
+        }
+    }
+    if constantExp != () {
+        words.push(")");
+    }
+    PointerType resultPtrType = pointerType(resultType, resultAddressSpace);
+    return [words, resultPtrType];
+}
+
+function bitCastBody(Value val, PointerType destTy, "constantExp"? constantExp=()) returns (string|Unnamed)[] {
+    (string|Unnamed)[] words = [];
+    words.push("bitcast");
+    if constantExp != () {
```

Comment:
These repeated `if constantExp` don't feel clean.  I think this would work better

- make the function be `bitCastArgs`
- do not have a `constantExp` argument
- pass `words` as the argument to bitCastArgs
- `bitCastArgs` constructs the part that needs to go in parentheses when it's const (i.e. not including `bitcast`)
- insert the parentheses in `constBitCast` arounds calls to `bitCastArgs`

Similarly for the other two.

Reply:
Fixed